docs/standard/cross-platform/index.md

```diff
@@ -25,7 +25,7 @@ You can develop apps for both Microsoft and non-Microsoft platforms by using the
+|Share source code between Windows Phone 8.1 and Windows 8.1 apps|**Shared projects** (Universal Apps template in Visual Studio 2013, Update 2).<br /><br /> -   Currently no Visual Basic support.<br />-   You can separate platform-specific code by using #`if` statements.<br /><br /> For details, see:<br /><br /> -   [Build apps that target Windows and Windows Phone by using Visual Studio](https://msdn.microsoft.com/library/windows/apps/dn609832.aspx) (MSDN article)<br />-   [Using Visual Studio to build Universal XAML Apps](http://blogs.msdn.com/b/visualstudio/archive/2014/04/14/using-visual-studio-to-build-universal-xaml-apps.aspx) (blog post)<br />-   [Using Visual Studio to Build XAML Converged Apps](https://channel9.msdn.com/Events/Build/2014/3-591) (video)|  
 |Share binaries between apps that target different platforms|**Portable Class Library projects** for code that is platform-agnostic.<br /><br /> -   This approach is typically used for code that implements business logic.<br />-   You can use Visual Basic or C#.<br />-   API support varies by platform.<br />-   Portable Class Library projects that target Windows 8.1 and Windows Phone 8.1 support Windows Runtime APIs and XAML. These features aren't available in older versions of the Portable Class Library.<br />-   If needed, you can abstract out platform-specific code by using interfaces or abstract classes.<br /><br /> For details, see:<br /><br /> -   [Portable Class Library](../../../docs/standard/cross-platform/cross-platform-development-with-the-portable-class-library.md) (MSDN article)<br />-   [How to Make Portable Class Libraries Work for You](http://blogs.msdn.com/b/dsplaisted/archive/2012/08/27/how-to-make-portable-class-libraries-work-for-you.aspx) (blog post)<br />-   [Using Portable Class Library with MVVM](../../../docs/standard/cross-platform/using-portable-class-library-with-model-view-view-model.md) (MSDN article)<br />-   [App Resources for Libraries That Target Multiple Platforms](../../../docs/standard/cross-platform/app-resources-for-libraries-that-target-multiple-platforms.md) (MSDN article)<br />-   [.NET Portability Analyzer](http://visualstudiogallery.msdn.microsoft.com/1177943e-cfb7-4822-a8a6-e56c7905292b) (Visual Studio extension)|  
```

Comment:
another one that can be changed to https if new url is used: https://blogs.msdn.microsoft.com/dsplaisted/2012/08/27/how-to-make-portable-class-libraries-work-for-you/